openam-clientsdk/dependency-reduced-pom.xml

```diff
@@ -67,7 +67,7 @@
         <configuration>
           <artifactSet>
             <includes>
-              <include>org.forgerock.commons:forgerock-util</include>
+              <include>org.forgerock.ce.commons:forgerock-util</include>

```

Comment:
Should all GroupIds should change from org.forgerock.<blah> to org.forgerock.ce.<blah>? I take it the shade plugin is pulling these artifacts and packaging them into a jar?